Added Roles and Organizationaal Contexts Mid-Level CCO Design Pattern

Tsimmons1234 opened the pull request:
Added roles and organizational contexts .md/.png to realizable entities folder.

giacomodecolle commented on design-patterns/new-patterns/realizable entities/roles/RolesAndOrganizationalContext.md:
```diff
@@ -0,0 +1,27 @@
+    ont00001262"]
+    A -- hasValue --> G["xsd string"]
+    A -- bearerOf_BFO0000196 --> H{"Role 1"}
+    H -- rdf:type --> B["Some Role"]
```
Can you take away the "some role" parts?

giacomodecolle commented on design-patterns/new-patterns/realizable entities/roles/RolesAndOrganizationalContext.md:
```diff
@@ -0,0 +1,27 @@
+flowchart BT
+    A{"Person 1"} -- rdf:type --> F["Person
+    ont00001262"]
+    A -- hasValue --> G["xsd string"]
```
Delete this "has value" and the string

giacomodecolle commented on design-patterns/new-patterns/realizable entities/roles/RolesAndOrganizationalContext.md:
```diff
@@ -0,0 +1,27 @@
+    H -- rdf:type --> B["Some Role"]
+    B -- rdf:type --> C["Role 
+    BFO0000023"]
+    H -- hasOrganiationalContext_ont00001992 --> K{"Organization 1"}
```
"organizational" typo

Tsimmons1234 commented:
- Removed "some role/some organization" component of design pattern
- Removed "has value" portion of design pattern
- Applied real world names to individual instances 
- Fixed typo in "hasOrganizationalContext"
